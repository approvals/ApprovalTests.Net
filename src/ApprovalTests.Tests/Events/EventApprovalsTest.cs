namespace ApprovalTests.Tests.Events
{
    using ApprovalTests.Events;
    using NUnit.Framework;

    [TestFixture]
    public class EventApprovalsTest
    {
        [Test]
        public void MulticastPoco()
        {
            var testingPoco = new TestingEventPoco();
            testingPoco.MyEvent += TestingListener.AnotherStandardCallback;
            testingPoco.MyEvent += TestingListener.StandardCallback;
            EventApprovals.VerifyEvents(testingPoco);
        }

        [Test]
        public void UnicastPoco()
        {
            var testingPoco = new TestingEventPoco();
            testingPoco.MyEvent += TestingListener.StandardCallback;
            testingPoco.PropertyChanged += TestingListener.PropertyChangedCallback;
            EventApprovals.VerifyEvents(testingPoco);
        }
    }
}