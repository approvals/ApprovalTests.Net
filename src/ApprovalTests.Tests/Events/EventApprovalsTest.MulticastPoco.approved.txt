Event Configuration for TestingEventPoco 

MyEvent:
	[0] Void AnotherStandardCallback(System.Object, System.EventArgs)
	[1] Void StandardCallback(System.Object, System.EventArgs)

